pythonforandroid/recipes/aiohttp/__init__.py

```diff
@@ -1,16 +1,17 @@
 """Build AIOHTTP"""
 from typing import List
-from pythonforandroid.recipe import CythonRecipe  # type: ignore
+from pythonforandroid.recipe import CppCompiledComponentsPythonRecipe
 
-
-class AIOHTTPRecipe(CythonRecipe):  # type: ignore # pylint: disable=R0903
-    """Build AIOHTTP"""
-
-    version = "v3.6.2"
-    url = "https://github.com/aio-libs/aiohttp/archive/{version}.zip"
+class AIOHTTPRecipe(CppCompiledComponentsPythonRecipe):  # type: ignore # pylint: disable=R0903
+    url = "https://files.pythonhosted.org/packages/ff/4f/62d9859b7d4e6dc32feda67815c5f5ab4421e6909e48cbc970b6a40d60b7/aiohttp-3.8.3.tar.gz"
```

Comment:
Can you update that line to leverage the version variable, and also use the github url?
Like so:
```python
version = "3.8.3"
url = "https://github.com/aio-libs/aiohttp/archive/v{version}.zip"
```